src/app/screens/BookDetail/index.tsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
-import { TouchableOpacity, Text } from 'react-native';
-import { NavigationProps } from '@interfaces/navigation';
 
-function BookDetail({ navigation }: NavigationProps) {
-  const handleGoBack = () => navigation.goBack();
+import { BOOKS } from '../Library/mockupData';
+
+import BookCard from './components/BookCard';
+
+function BookDetail() {
+  const { imageUrl, title, author, genre, year, isAvailable } = BOOKS[3];
 
   return (
-    <TouchableOpacity onPress={handleGoBack}>
-      <Text>Go Back</Text>
-    </TouchableOpacity>
+    <BookCard
+      imageUrl={imageUrl}
+      title={title}
+      author={author}
+      genre={genre}
+      year={year}
+      isAvailable={isAvailable}
+    />
   );
 }
```

Comment:
⚠️ This is wrong! You must receive via navigation param the book not hard-code it
```
function BookDetail({ navigation }: Props) {
const book = navigation.getParam('book', {});
...
return <BookCard {...book} />;
}

In the Library/BookList => `const onPress = () => navigation.navigate(BookDetail,  { book });`